code/game/objects/structures/crates_lockers/closets/secure/genpop.dm

```diff
@@ -0,0 +1,97 @@
+/obj/structure/closet/secure_closet/genpop
+	desc = "It's a secure locker for inmates's personal effects."
+	var/default_desc = "It's a secure locker for the storage inmates's personal effects during their time in prison."

```

Comment:
Can we change `effects` to `belongings`? Overly fancy word, non-native speakers will likely have trouble understanding.